v1/api/swagger/components/schemas/Limits.yaml

```diff
@@ -1,4 +1,9 @@
 type: object
+required:
+  - Hosts
+  - Volumes
+  - VolumeCapacity
+  - PrivateNetworks
```

Comment:
Are these "required"?  If not present, they'd simply considered as 0, or unlimited.

Reply:
They are considered required for the project [here](https://github.com/HewlettPackard/hpegl-metal-client/blob/master/v1/api/swagger/components/schemas/Project.yaml#L6), I've included this section because the machine size limits are not. See current spec 
![image](https://user-images.githubusercontent.com/69802962/185505148-55f44b80-678f-43b3-beb2-621057bfa749.png)